easyrest/routes.py

```diff
@@ -1,3 +1,7 @@
 def includeme(config):
     config.add_static_view('static', 'static', cache_max_age=3600)
     config.add_route('home', '/')
+    # RazSK 28.12.2018 Begin
+    # add routes for Restaurant
+    config.add_route('restaurant', 'restaurants/{id}', request_method="GET")
```

Comment:
I'd rather use 2 routes:
```
config.add_route('restaurants_all', 'restaurants', request_method="GET")
config.add_route('restaurant', 'restaurants/{id}', request_method="GET")
```
First for all restaurants, second - specified one.

And each route would have a separate view.